plugins/rxp/test/RxpReaderTest.cpp

```diff
@@ -131,7 +130,7 @@ TEST(RxpReaderTest, testRead)
 TEST(RxpReaderTest, testNoPpsSync)
 {
     Options options = defaultRxpReaderOptions();
-    Option syncToPps("sync_to_pps", "false", "");
+    Option syncToPps("sync_to_pps", false);
```

Comment:
@abellgithub is this still the accepted way of setting options on a `Reader`? My `sync_to_pps` argument (setting [this option](https://github.com/PDAL/PDAL/pull/1257/files#diff-f95fa921d6512b222740204d336a20faR87)) isn't getting picked up by my reader in this test, and I haven't been able to track down at what point in the option processing this option's value is getting discarded. If this is still the accepted way of setting options, then the problem must be between my keyboard and chair.